discovery-provider/src/tasks/index_nethermind.py

```diff
@@ -930,9 +948,31 @@ def revert_blocks(self, db, revert_blocks_list):
                 )
                 if previous_track_route_entry:
                     previous_track_route_entry.is_current = True
-                logger.info(f"Reverting track route {track_route_to_revert}")
+                logger.info(
+                    f"index_nethermind.py | Reverting track route {track_route_to_revert}"
+                )
                 session.delete(track_route_to_revert)
 
+            for playlist_route_to_revert in revert_playlist_routes:
+                playlist_id = playlist_route_to_revert.playlist_id
+                previous_track_route_entry = (
+                    session.query(PlaylistRoute)
+                    .filter(
+                        PlaylistRoute.track_id == track_id,
```

Comment:
i think this wont work? should be `PlaylistRoute.playlist_id == playlist_id` i believe